tools/generate-custom-icon-webfont

```diff
@@ -34,6 +34,6 @@ fs.readdir('./static/assets/icons', function (err, files) {
             console.error(err);
             process.exit(1);
         }
-        console.log('SUCCESS! Webfonts generated in /static/icons/fonts.');
+        console.log('SUCCESS! Webfonts generated in /static/generated/icons/fonts.');
```

Comment:
Should be ./static/generated/icons/fonts

Reply:
Addressed.